activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+*   Fixed inconsistency between `first(n)` and `last(n)` when
+    used with `limit()` The `first` finder respects the `limit()`,
+    making it consistent with `last(n)`.
```

Comment:
I think that we should note fixing inconsistency `first(n)` with either loaded or not.
Previously, we need to use `relation.to_a.first(n)` as a workaround, but now, we can use `relation.first(n)` simply.